extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -53,10 +53,10 @@
    z-index: 2147483646;
    width: 100%;
    position: fixed;
-   top: 0;
+   background-color: white;
  }
 
- .i-amphtml-toolbar > ul {
+ .i-amphtml-toolbar > ul:first-child {
```

Comment:
That would target the elements that are not the first child of our \<nav toolbar\> . 

Oh wait I was incorrect: https://developer.mozilla.org/en-US/docs/Web/CSS/Child_selectors

It may have been a mistmatch in classes from the AMP start heme, I'll play with it and see what happens